stencil/src/components/pulumi-tertiary-nav/pulumi-tertiary-nav.tsx

```diff
@@ -0,0 +1,64 @@
+import { Component, Host, h, Prop } from "@stencil/core";
+
+export interface TertiaryNavItem {
+  // The anchor for the content that the nav item controls viewability of
+  anchor: string;
+  // The text that should show for the nav item
+  label: string;
+}
+
+@Component({
+    tag: "pulumi-tertiary-nav",
+    styleUrl: "pulumi-tertiary-nav.css",
+    shadow: false,
+})
+export class PulumiTertiaryNav {
+    // TertiaryNavItem[], passed into the component as stringified JSON
+    @Prop({ mutable: true })
+    items: string;
+
+    @Prop({ mutable: true })
+    parsedItems: TertiaryNavItem[];
+
+    @Prop({ mutable: true })
+    tabContent: string[];
+
+    // The currently selected tab.
+    @Prop({ mutable: true })
+    selection: any;
+
+    componentWillLoad() {
+        this.parsedItems = JSON.parse(this.items);
+
+        // Collection of anchor tags
+        this.tabContent = this.parsedItems.map((item) => item.anchor);
+
+        // The first tab is selected on initial load.
+        this.selectTab(this.parsedItems[0]);
+    }
+
+    private selectTab(item: TertiaryNavItem) {
+        this.selection = item.anchor;
+
+        $(`#${item.anchor}`).removeClass("hidden");
```

Comment:
I almost missed that we're referencing jQuery here. Should we be, though? Seems like ideally we'd want _not_ to rely on jQuery within a component like this.

Reply:
I removed the jQuery and replaced it with `document.getElementById("element").classList.add('class');`, to continue the pattern of keeping jquery in the `/src/ts/` files in the repo.  I also think this relates to some of your bigger picture questions in the hugo PR, so I'll answer those there.

Reply:
Great, thanks!